R/library_call_linter.R

```diff
@@ -1,46 +1,63 @@
 #' Library call linter
 #'
-#' Force library calls to all be at the top of the script.
+#' This linter covers several rules related to [library()] calls:
+#'
+#'  - Enforce such calls to all be at the top of the script.
+#'  - Block usage of argument `character.only`.
```

Comment:
Maybe also add rule about making library calls directly over vectorized calls?

Reply:
Thanks. Will revisit once this is merged to `library_call_linter()`

Reply:
Oops, memory failed me :)